src/ui/controls/edit.cpp

```diff
@@ -1439,414 +1455,420 @@ void CEdit::FreeImage()
 
 bool CEdit::ReadText(std::string filename, int addSize)
 {
-    int         len, i, j, n, font, iLines, iCount;
```

Comment:
You did something weird to whitespace in this whole block (and a few other ones) but I can't quite figure out what. Indentation seems correct, line endings possibly?